test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckDuplicateDeclarationDetailsControllerSpec.scala

```diff
@@ -187,24 +192,46 @@ class CheckDuplicateDeclarationDetailsControllerSpec
     }
 
     "handle submit requests" when {
+
+      def performAction(journey: JourneyBindable, data: Seq[(String, String)]): Future[Result] =
+        controller.submit(journey)(FakeRequest().withFormUrlEncodedBody(data: _*))
+
+      //TODO Update Redirect url when decision is made
       "the user confirms the duplicate details are correct" in forAll(journeys) { journey =>
-        def performAction(data: Seq[(String, String)]): Future[Result] =
-          controller.submit(journey)(FakeRequest().withFormUrlEncodedBody(data: _*))
-        val displayDeclaration                                         = sample[DisplayDeclaration]
+        val displayDeclaration = sample[DisplayDeclaration]
+        val session            = sessionWithClaimStateForDuplicate(Some(displayDeclaration), Some(toSelectNumberOfClaims(journey)))
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(session._1)
+        }
 
-        val session = sessionWithClaimStateForDuplicate(Some(displayDeclaration))
+        checkIsRedirect(
+          performAction(journey, Seq(checkDeclarationDetailsKey -> "0")),
+          routes.EnterCommoditiesDetailsController.enterCommoditiesDetails(journey)
+        )
+      }
+
+      //TODO Update Redirect url when decision is made
+      "the user confirms the details are incorrect" in forAll(journeys) { journey =>
+        val displayDeclaration = sample[DisplayDeclaration]
+        val session            = sessionWithClaimStateForDuplicate(Some(displayDeclaration), Some(toSelectNumberOfClaims(journey)))
 
         inSequence {
           mockAuthWithNoRetrievals()
           mockGetSession(session._1)
         }
 
         checkIsRedirect(
-          performAction(Seq("check-declaration-details" -> "0")),
-          routes.EnterCommoditiesDetailsController.enterCommoditiesDetails(JourneyBindable.Single)
+          performAction(journey, Seq(checkDeclarationDetailsKey -> "1")),
+          routes.EnterCommoditiesDetailsController.enterCommoditiesDetails(journey)
         )
       }
 
+      //TODO Write testcase for: User submits no answer
+
+      //TODO Write testcase for: User submits an incorrect answer
```

Comment:
do you still need to do these or can these comments be removed?

Reply:
There are 3 TODOs in this file.
This is related to the Duplicate Declaration Details Page, where we ignore the user input and just go to the next page.
I added these TODOs here, so that when the business decides what to do with the input, then I don't forget to add the testcases.
Unfortunately I cannot check for wrong or error input, when we don't validate the input.